README.md

```diff
@@ -1,11 +1,45 @@
 TRAPpy [![Build Status](https://travis-ci.org/ARM-software/trappy.svg?branch=master)](https://travis-ci.org/ARM-software/trappy)
 ======
 
-trappy is a visualization tool to help analyze data generated on a
-device. It depends on ipython notebook and pandas. First install these
-dependencies if you don't have them already:
+TRAPpy (Trace Analysis and Plotting in Python) is a visualization tool to help
+analyze data generated on a device. It parses ftrace-like logs and creates
+in-memory data structures to be used for plotting and data analysis.
 
-    $ apt-get install ipython-notebook python-pandas
+# Installaion
+
+## Required dependencies
+
+##### Install additional tools required for some tests and functionalities
+
+	$ sudo apt-get install trace-cmd kernelshark
+
+##### Install the Python package manager
+
+	$ sudo apt-get install python-pip python-dev
+
+##### Install required python packages
+
+	$ sudo apt-get install libfreetype6-dev libpng12-dev python-nose
+	$ sudo pip install numpy matplotlib pandas ipython[all]
+
+## Clone the repository
```

Comment:
And leave this for developers.  Then again, if you are a developer you should know how to clone the repository so maybe this is redundant.